<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectionRectDialog</class>
 <widget class="QDialog" name="InspectionRectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inspection Window</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>118</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelReferenceImage">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>118</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QFrame" name="frameDashBoardBlockName">
       <property name="styleSheet">
        <string notr="true">QFrame{
     background:rgb(249,247,248);

}

QLabel{
  background:rgb(249,247,248);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}

QRadioButton{
     background:rgb(255,255,255);
      color: rgb(39,51,66);
     font-size: 16px;
     font-weight: bold;
     font-family: Siemens Sans;
     min-height: 32px;
     max-height: 32px;
}

QRadioButton::indicator {
width: 15px;
height: 15px;
border-radius: 15px;
}

QRadioButton::indicator::unchecked{ 
border: 3px solid; 
border-color: rgb(196,199,204);
border-radius: 9px;
background-color: white; 
width: 13px; 
height: 13px; 
}

QRadioButton::indicator::checked{ 
border: 3px solid; 
border-color: rgb(196,199,204);
border-radius: 9px;
background-color: rgb(0,95,135); 
width: 13px; 
height:13px; 
}






</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="labelDashBoardInspectionWindow">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string> Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditInspectionWindowName">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
   color: rgb(39,51,66);
   font-size: 16px;
   font-weight: bold;
   font-family: Siemens Sans;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QFrame" name="frameDashBoardPrintTolerance">
       <property name="styleSheet">
        <string notr="true">QFrame{
     background:rgb(249,247,248);

}

QLabel{
  background:rgb(249,247,248);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}

QRadioButton{
     background:rgb(255,255,255);
      color: rgb(39,51,66);
     font-size: 16px;
     font-weight: bold;
     font-family: Siemens Sans;
     min-height: 32px;
     max-height: 32px;
}

QRadioButton::indicator {
width: 15px;
height: 15px;
border-radius: 15px;
}

QRadioButton::indicator::unchecked{ 
border: 3px solid; 
border-color: rgb(196,199,204);
border-radius: 9px;
background-color: white; 
width: 13px; 
height: 13px; 
}

QRadioButton::indicator::checked{ 
border: 3px solid; 
border-color: rgb(196,199,204);
border-radius: 9px;
background-color: rgb(0,95,135); 
width: 13px; 
height:13px; 
}






</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelDashBoardPrintTolerance">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string> Print Tolerance</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxPrintErrorTolInPercent">
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QDoubleSpinBox{
     background:rgb(255,255,255);
     color:rgb(39,51,66);
}</string>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>30.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QFrame" name="frameDashBoardPrintProperty">
       <property name="styleSheet">
        <string notr="true">QFrame{
     background:rgb(249,247,248);

}

QLabel{
  background:rgb(249,247,248);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}

QRadioButton{
     background:rgb(255,255,255);
      color: rgb(39,51,66);
     font-size: 16px;
     font-weight: bold;
     font-family: Siemens Sans;
     min-height: 32px;
     max-height: 32px;
}

QRadioButton::indicator {
width: 15px;
height: 15px;
border-radius: 15px;
}

QRadioButton::indicator::unchecked{ 
border: 3px solid; 
border-color: rgb(196,199,204);
border-radius: 9px;
background-color: white; 
width: 13px; 
height: 13px; 
}

QRadioButton::indicator::checked{ 
border: 3px solid; 
border-color: rgb(196,199,204);
border-radius: 9px;
background-color: rgb(0,95,135); 
width: 13px; 
height:13px; 
}






</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_35">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_36">
          <item row="2" column="0">
           <widget class="QRadioButton" name="radioButtonPrintPropertyDynamic">
            <property name="font">
             <font>
              <family>Siemens Sans</family>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Dynamic</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButtonPrintPropertyStatic">
            <property name="font">
             <font>
              <family>Siemens Sans</family>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Static</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelDashBoardPrintProperty">
            <property name="font">
             <font>
              <family>Siemens Sans</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Print Property</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="labelOnlyLineCheck">
           <property name="text">
            <string>Line Check On</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="checkBoxUseOnlyLineCheck">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="labelNumberLines">
           <property name="text">
            <string>Number Lines</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxNumberLines">
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QFrame" name="frameDashBoardBlockHeight">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>82</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
     background:rgb(249,247,248);
    
     min-height: 80px;
     max-height: 80px;
}

QLabel{
  background:rgb(249,247,248);
  border: 0px solid rgb(0,255,0);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}

QDoubleSpinBox{
background:rgb(255,255,255);
  color: rgb(39,51,66);
  font-size: 16px;
  font-weight: bold;
  font-family: Siemens Sans;
  min-height: 32px;
  max-height: 32px;
}

QDoubleSpinBox::focus{
 border: 2px solid rgb(0,95,135);
}

</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_20">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelDashBoardROIROIHoseDetectionHeight">
            <property name="font">
             <font>
              <family>Siemens Sans</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Block Height</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxBlockHeight">
            <property name="font">
             <font>
              <family>Siemens Sans</family>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QDoubleSpinBox{
     background:rgb(255,255,255);
     color:rgb(39,51,66);
}</string>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string> mm</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>0.500000000000000</double>
            </property>
            <property name="maximum">
             <double>500.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>2.500000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QFrame" name="frameDashBoardBlockWidth">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>82</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
     background:rgb(249,247,248);
    
     min-height: 80px;
     max-height: 80px;
}

QLabel{
  background:rgb(249,247,248);
  border: 0px solid rgb(0,255,0);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}

QDoubleSpinBox{
background:rgb(255,255,255);
  color: rgb(39,51,66);
  font-size: 16px;
  font-weight: bold;
  font-family: Siemens Sans;
  min-height: 32px;
  max-height: 32px;
}

QDoubleSpinBox::focus{
 border: 2px solid rgb(0,95,135);
}

</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_19">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelDashBoardBlockWidth">
            <property name="font">
             <font>
              <family>Siemens Sans</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Block Width</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxBlockWidth">
            <property name="font">
             <font>
              <family>Siemens Sans</family>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QDoubleSpinBox{
     background:rgb(255,255,255);
     color:rgb(39,51,66);
}</string>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string> mm</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>0.500000000000000</double>
            </property>
            <property name="maximum">
             <double>500.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>2.500000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QLabel" name="labelDisableBlock">
       <property name="text">
        <string>Disable Block</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBoxDisableBlock">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxUseOnlyLineCheck</tabstop>
  <tabstop>doubleSpinBoxNumberLines</tabstop>
  <tabstop>radioButtonPrintPropertyStatic</tabstop>
  <tabstop>radioButtonPrintPropertyDynamic</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
